scripts/lint-examples/src/lintExamples.js

```diff
@@ -11,41 +10,70 @@
 const {execSync} = require('child_process');
 const {promises: fs} = require('fs');
 const path = require('path');
-
 const glob = require('glob-promise');
 
 /**
  * The root document to search for documents
  */
-const documentsRoot = path.join(__dirname, '..', '..', 'docs');
+const documentsRoot = path.join(__dirname, '..', '..', '..', 'docs');
 
 /**
  * The directory to output generated files to
  */
-const outputRoot = path.join(__dirname, 'out');
+const outputRoot = path.join(__dirname, '..', 'out');
 
-const args = process.argv.slice(2);
+/**
+ * Process arguments to be forwarded to the linter
+ */
+const processArgs = process.argv.slice(2);
 
 /**
- * Main function
+ * Valid extensions for snack examples
  */
-(async () => {
+const validExtensions = ['js', 'tsx'];
```

Comment:
It seems kind of odd to use `tsx`, but not `jsx`, but Snack seems to not support `jsx`, and the RN 0.71 ESLint config also no-oped on JSX. So instead we check that it is either `js` or `tsx` to avoid the inevitable confusion if someone adds `jsx` or `ts`.